Mathlib/GroupTheory/SpecificGroups/Cyclic.lean

```diff
@@ -653,4 +653,32 @@ protected theorem ZMod.exponent (n : ℕ) : AddMonoid.exponent (ZMod n) = n := b
   · rw [IsAddCyclic.exponent_eq_zero_of_infinite]
   · rw [IsAddCyclic.exponent_eq_card, card]
 
+/-- A group of order `p ^ 2` is not cyclic if and only if its exponent is `p`. -/
+@[to_additive]
```

Comment:
If all goes as planned, #8722 will make `not_isAddCyclic_iff_exponent_eq_prime` the `to_additivized` name of this lemma!

Reply:
Tao's PFR project introduces an [ElementaryAddCommGroup](https://github.com/teorth/pfr/blob/245ee1eb12a39ba36299812f0ba11557b673fc41/PFR/f2_vec.lean#L21-L22) class, and it would be nice to show it [must be the case](https://math.stackexchange.com/questions/212084/group-with-order-p2-must-be-abelian-how-to-prove-that) here. cc @tb65536

Reply:
@alreadydone, moogle suggests docs#IsPGroup.commGroupOfCardEqPrimeSq.

Reply:
Nice, thanks!

Reply:
The `to_additive` PR has been merged: if you merge the new master, this error should disappear!